src/controllers/TreeController.php

```diff
@@ -197,4 +197,4 @@ protected function getModel($paramName = null, $createIfNotFound = false)
 
         return $model;
     }
-}
+}
```

Comment:
It wasn't included in your first commit. Revert this changes via history or restore file to original condition and apply your fixes again.